job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy

```diff
@@ -39,6 +39,7 @@ class Constants {
     static final String REPORT_BRANCH = '7.x'
     static final String BUILD_DATE = ''
     static final String MAIN_BRANCH_PROD_VERSION = '7.14.0'
+    static final String KOGITO_MAIN_BRANCH_PROD_VERSION = '1.14.0'
```

Comment:
is this correct? i think main branch is more forward

Reply:
so there is already 1.19.x branch and main is probably targeting now 1.20.x

Reply:
I see `2.0.0-SNAPSHOT` on main: https://github.com/kiegroup/kogito-runtimes/blob/main/pom.xml#L15 - not sure how it translates into the product, though.

Reply:
well, the snapshot on main is make up to not upgrade it in every release ;)

Reply:
Updated, thanks!